src/Http/Http.Extensions/test/RequestDelegateGenerator/Baselines/MapAction_ExplicitBodyParam_ComplexReturn_Snapshot.generated.txt

```diff
@@ -241,7 +250,10 @@ namespace Microsoft.AspNetCore.Http.Generated
                             httpContext.Response.StatusCode = 400;
                         }
                         var result = await filteredInvocation(EndpointFilterInvocationContext.Create<global::Microsoft.AspNetCore.Http.Generators.Tests.Todo?>(httpContext, todo_local));
-                        await GeneratedRouteBuilderExtensionsCore.ExecuteObjectResult(result, httpContext);
+                        if (result is not null)
```

Comment:
What happens if the filter returns `null`? What gets written to the Response?

Reply:
The response remains untouched. This matches the behavior of the filter construction pipeline that the runtime uses ([ref](https://github.com/dotnet/aspnetcore/blob/6575b289081b9f85fb30d01c7b306d1ec5d00a68/src/Http/Routing/src/RequestDelegateFilterPipelineBuilder.cs#L56-L61)).